mirdata/datasets/beatles.py

```diff
@@ -145,149 +148,115 @@ def to_jams(self):
         )
 
 
-def load_audio(audio_path):
+@io.coerce_to_bytes_io
+def load_audio(fhandle: BinaryIO) -> Tuple[np.ndarray, float]:
     """Load a Beatles audio file.
 
     Args:
-        audio_path (str): path to audio file
+        fhandle: path or file-like object pointing to an audio file
 
     Returns:
-        * np.ndarray - the mono audio signal
-        * float - The sample rate of the audio file
+        * np.ndarray: the mono audio signal
```

Comment:
This is still not rendering properly.

![image](https://user-images.githubusercontent.com/8169843/104616295-1e46cf80-5658-11eb-8682-bcb985159a60.png)

Can you keep them as they were?

Reply:
Ah rats I messed up. I'll change it to dash